src/utils/context_utils.py

```diff
@@ -161,10 +161,9 @@ def get_server_settings(
     )
 
     if _SERVER_SETTINGS_KEY in context.metadata:
-        # logging.info(
-        #     f"Getting CACHED server_settings from workspace {context.client.config.workspace_handle}.",
-        # )
-
+        logging.info(
+            f"Getting CACHED server_settings from workspace {context.client.config.workspace_handle}.",
+        )
```

Comment:
Did you mean to leave these logging statements in? I don't have an opinion on whether they should or shouldn't be